symbolicationWebService.py

```diff
@@ -222,32 +238,35 @@ def ReadConfigFile():
   return True
 
 def Main():
-  global gSymFileManager, gOptions
+  global gSymFileManager, gOptions, gPool
 
   if not ReadConfigFile():
     return 1
 
-  SetLoggingOptions(gOptions["Log"])
+  # In a perfect world, we could create a process per cpu core.
+  # But then we'd have to deal with cache sharing
+  gPool = Pool(1)
+  gPool.submit(initializeSubprocess, gOptions)
 
-  # Create the .SYM cache manager singleton
-  gSymFileManager = SymFileManager(gOptions)
-
-  # Prefetch recent symbols
-  gSymFileManager.PrefetchRecentSymbolFiles()
+  SetLoggingOptions(gOptions["Log"])
```

Comment:
This is to initialize logging in the child process. I added some comments about that.